### ISSUE TITLE
Visually show whether shuffle is active or not

### ISSUE THREAD

**Issue by tavareshenrique** [getomni/spicetify#6]
I do not know if it is a bug or if the initial idea of the project was not to use this information, so I opened this PR as an "Enhancement request".

I think it's cool to know if the Shuffle option is checked or not, how it works on Spotify:

![spotify2](https://user-images.githubusercontent.com/27022914/115275344-0c257300-a118-11eb-8d1e-f9566dc7eb3a.png)

To find out if Shuffle is active or not in Spicetify theme I have to hover over it and check this information, like this:

![deepin-screen-recorder_Select area_20210419140052](https://user-images.githubusercontent.com/27022914/115275330-06c82880-a118-11eb-8190-c04c9af0b42e.gif)

Maybe if he looked like this, it would be visually cool:

![spotify1](https://user-images.githubusercontent.com/27022914/115275360-134c8100-a118-11eb-9d8f-32196d6d0fa5.png)

It's just a suggestion. Congratulations for the great job, I loved it. 💜

**Comment by jpedroschmitz** [comment]
Yeah, we should add an active state when the shuffle option is turned on.

@birobirobiro, when you have time, let me know if you can take this issue =D
